nestkernel/nest_names.cpp

```diff
@@ -364,6 +367,9 @@ const Name withgid( "withgid" );
 const Name withpath( "withpath" );
 const Name withtime( "withtime" );
 const Name withweight( "withweight" );
+const Name withreceivergid( "withreceivergid" );
```

Comment:
As the parameter for the sender is called `withgid` I chose to call this parameter `withtargetgid`. I think this is more consistent.